labs/2-perpetual-exchange/3-jlp-pool.md

```diff
@@ -126,3 +130,7 @@ revenue share you generate = $35,000 x 0.025% = $8.75
 ```
 
 Every Monday, at ~ UTC 18:00, the Estimated APY figure is updated with the above calculation, by using the previous week's fees and estimating an APR and APY for users.
+
+### Recommended action during bull market
+
+JLP is denominated in USD value. During bull market, JLP will unlikely to outperform SOL, ETH or BTC. It is recommended to swap stable token into JLP and earn the yield by trading activities.
```

Comment:
just put this under Risk. no need a new section.

Reply:
```
JLP is denominated in USD. During bull market, JLP is unlikely to outperform SOL, ETH or BTC. It is recommended to swap stable token (USDC/USDT) into JLP and earn the yield from the perps trading activities.
```

Reply:
Recommended for whom? This language reads like financial advice to me.